StreamChatClient_v3/Workers/ChannelUpdater.swift

```diff
@@ -19,4 +18,31 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
             }
         }
     }
+
+    func deleteChannel(cid: ChannelId, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .deleteChannel(cid: cid)) { result in
+            switch result {
+            case .success: completion?(nil)
+            case let .failure(error): completion?(error)
+            }
```

Comment:
This is so common maybe we should create a helper for that in a follow up PR. What do you think?

Reply:
Yes, that makes sense